controller/internal/routes/config_router.go

```diff
@@ -119,6 +122,6 @@ func (r *ConfigRouter) Patch(ae *env.AppEnv, rc *response.RequestContext, params
 		if err != nil {
 			return err
 		}
-		return ae.Handlers.Config.Patch(model, fields.FilterMaps("tags", "data"))
+		return ae.Managers.Config.Update(model, fields.FilterMaps("tags", "data"))
```

Comment:
just checking that this was an intentional change. I don't know the differences tween patch/update but this seems like it was a bug that you found/fixed along the way?

Reply:
Only difference between update and patch is that patch allowed a FieldChecker (to specify which fields had changed). Now Update takes a field checker and you can pass in nil if you want to update all fields. This style works a bit better with the update command we have which includes an optional updated fields field.